Header header

int8 offset
uint8 special_case